blog-post.html

```diff
@@ -1,12 +1,138 @@
-<!DOCTYPE html>
+<!DOCTYPE HTML>
+<link href="blog.css" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Peddana&display=swap" rel="stylesheet">
+<script src="https://kit.fontawesome.com/yourcode.js"></script>
+
 <html>
-  <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+  <!--post only title -->
+  <div class="container">
+    <div class="headerbox">
+      <!--Title Home and underscipt-->
+      <a href="index.html" class="blogtitle"> BLOGZ</a>
+      <h2 class="undertitle">
+        <p> One Of the Top Ten Blogs</p>
+      </h2>
+      <hr class="linebreak">
+      <hr class="line2">
+    </div>
+    <h2 class="blogtext"> The Best Blog You've Ever Seen</h2>
+    <main>
+      <img class="blogimage" img
+        src="https://static01.nyt.com/images/2020/04/30/arts/29river-notebook2/29river-notebook2-videoSixteenByNineJumbo1600.jpg">
+      <!--Blog Content, Paragraph Collection-->
+      <h2 class="content"> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Iceland swag

```

Comment:
using `&nbsp` is this fashion creates quite a clunky way of moving things around on the page. you could accomplish the same thing by using your `flexbox` and `justify-content`

Doing it with `&nbsp` wouldn't adjust depending on the size of the monitor